packages/http/src/forwarder/__tests__/HttpForwarder.spec.ts

```diff
@@ -7,20 +7,21 @@ describe('HttpForwarder', () => {
   const forwarder = new HttpForwarder();
 
   beforeEach(() => {
-    jest.spyOn(axios, 'default').mockImplementation(jest.fn);
+    jest.spyOn(axios, 'default');
   });
 
   describe('forward()', () => {
     describe("parameters haven' been provided", () => {
       it('proxies request correctly', async () => {
-        jest.spyOn(axios, 'default').mockImplementation(() => ({
+        jest.spyOn(axios, 'default').mockResolvedValue({
```

Comment:
Is that the test that should return promise but returned actual value that you mentioned in Slack?
Looking at the "expect" I'm not surprised it passed... It's not really resting the result, just the fact that it calls `default`

Reply:
It's still incorrect though, right?